### PR TITLE
 commit 11.02.2019 - 2

### PR DESCRIPTION
**Pull request by MateAcademy**

### CODE REVIEW

File: src/main/resources/Untitled4.sql [hunk 1]
```diff
@@ -0,0 +1,519 @@
+--Homework 11.02.2019
+SELECT
+    COUNT(*),
```
Comment (SerhiiPasko) [inline src/main/resources/Untitled4.sql:3]: What is it?

**Review by SerhiiPasko — APPROVED** [review]
Strange first and second requests

File: src/main/resources/Untitled4.sql [hunk 2]
```diff
@@ -0,0 +1,519 @@
...
+    pr.mfr_id;
+
+/* 7. Вивести імя компанії, скільки вона тратила в середньому на покупки товарів виробників, котрі виготовляють більше 4-ьох товарів */
+
+SELECT
+    c.cust_num,
+    c.company,
+    AVG(s.sales) AS avg_sales,
+    o.mfr
+FROM
```
Comment (DadizhaO) [inline src/main/resources/Untitled4.sql:347]: s.sales this is a field which means all total sales, i think better to use AMOUNT from ORDERS

File: src/main/resources/Untitled4.sql [hunk 3]
```diff
@@ -0,0 +1,519 @@
...
+        GROUP BY
+            mfr_id
+        HAVING
+            COUNT(product_id) > 4
+    ) mfr_more_4_prod ON orders.mfr = mfr_more_4_prod.mfr_id
+GROUP BY
+    customers.cust_num,
+    customers.company,
+    orders.mfr;
+
```
Comment (DadizhaO) [inline src/main/resources/Untitled4.sql:366]: this is the same as the previous, what is the difference?

**Review by DadizhaO — APPROVED** [review]

File: src/main/resources/Untitled4.sql (hunk 1 above)
Comment (MateAcademy) [reply src/main/resources/Untitled4.sql:3]: This is first  task on the page 33!

File: src/main/resources/Untitled4.sql (hunk 3 above)
Comment (MateAcademy) [reply src/main/resources/Untitled4.sql:366]: I will dell it

File: src/main/resources/Untitled4.sql (hunk 1 above)
Comment (MateAcademy) [reply src/main/resources/Untitled4.sql:3]: COUNT(*), - This is a command to calculate all values that meet the conditions.

**Pull request closed without merging**